lib/Locale/Country/Extra.pm

```diff
@@ -33,7 +33,17 @@ sub code_from_country {
     my ($self, $country) = @_;
 
     my %code_countries = reverse %{ $self->_country_codes };
-    return lc $code_countries{$country};
+
+
+    unless( exists $code_countries{$country} ){
+        use List::Util qw(first) ;
+        $country = lc $country;
+        $country = first { $country eq lc $_ } keys %code_countries;
+        return '' unless $country;
```

Comment:
That changed the behavior of function. `return undef` will be better.

Reply:
or `$country = (first { $country eq lc $_ } keys %code_countries) // '';` don't return here.

Reply:
// '' will not do here, because  lc undef will still trigger warning .
and the previous version of the fucntion will also trigger the warning.

return '' to keep consistent with other function in the moudle with also reutrn ''

Reply:
Sorry I still can't understand. why not return undef here ? The old version returned `undef` . Is there a special reason to change its API ?